<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupOptionsWidget</class>
 <widget class="QWidget" name="StartupOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Startup Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="boxTestList">
     <property name="title">
      <string>Test List  </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbStartupEmptyList">
        <property name="text">
         <string>use empty list after startup</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbnStartupLoadList">
          <property name="text">
           <string>load specific HML file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editStartupFileName"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnStartupFileDlg">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
          <property name="icon">
           <iconset resource="../../qHostMonitor.qrc">
            <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupRestoreList">
        <property name="text">
         <string>restore last status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="boxMonitoring">
     <property name="title">
      <string> Monitoring  </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QRadioButton" name="rbStartupMonitoringStart">
        <property name="text">
         <string>start after loading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupMonitoringStop">
        <property name="text">
         <string>stop after loading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupMonitoringRestore">
        <property name="text">
         <string>restore last status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="boxErrorHandler">
     <property name="title">
      <string> Error handler  </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbErrorNormal">
        <property name="text">
         <string>Normal start: show error msg (if any) and continue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbErrorIgnore">
        <property name="text">
         <string>Ignore errors during startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbErrorQuit">
        <property name="text">
         <string>Quit if any error occurs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="boxWindowsStatus">
     <property name="title">
      <string> Window status  </string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbStartupWindowMin">
        <property name="text">
         <string>minimize after loading</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupWindowMax">
        <property name="text">
         <string>maximize after loading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupWindowNormal">
        <property name="text">
         <string>normal size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStartupWindowRestore">
        <property name="text">
         <string>restore last status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QCheckBox" name="StartupChk">
     <property name="text">
      <string>Place program into the Startup group</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
